test/box-luatest/gh_10508_iproto_insert_arrow_test.lua

```diff
@@ -107,6 +119,12 @@ g.test_iproto_insert_arrow_invalid = function(cg)
             000004000000f0ffffff4000000001000000610000000600080004000c0010000400
             080009000c000c000c0000000400000008000a000c00040006000800ffffffff]])
         t.assert_equals(r[box.iproto.key.ERROR_24],
+                        'Invalid MsgPack - packet body')
+        t.assert_equals(r[box.iproto.key.ERROR][0][2][3],
+                        'Invalid MsgPack - invalid extension')
+        t.assert_equals(r[box.iproto.key.ERROR][0][3][3],
+                        'Invalid MsgPack - cannot unpack arrow data')
+        t.assert_equals(r[box.iproto.key.ERROR][0][4][3],
                         'ArrowArrayStreamGetNext failed: ' ..
                         'Expected at least 8 bytes in remainder of stream')
 
```

Comment:
Let's split this patch in 2:
1. Add MsgPack arrow extension validation.
2. All the rest.

The thing is we introduce here Lua arrow wrapper but test it only in the next patch. Let's merge 2 patch from the split with the next patch.

Reply:
Done.